pkgs/by-name/gv/gvmd/package.nix

```diff
@@ -0,0 +1,155 @@
+{
+  cmake,
+  doxygen,
+  fetchFromGitHub,
+  git,
+  glib,
+  gnutls,
+  gpgme,
+  graphviz,
+  gvm-libs,
+  icu,
+  lib,
+  libbsd,
+  libical,
+  libidn2,
+  libpkgconf,
+  libselinux,
+  libsepol,
+  libtasn1,
+  libxslt,
+  openvas-scanner,
+  ospd-openvas,
+  p11-kit,
+  perl538Packages,
+  pcre2,
+  pkg-config,
+  postgresql,
+  stdenv,
+  util-linux,
+  xmltoman,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "gvmd";
+  version = "23.4.0";
+
+  src = fetchFromGitHub {
+    owner = "greenbone";
+    repo = "gvmd";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-Bxq6o98viMMwt2eoiZgPDTYqD/z6jRWYLgH/ZJvkpV4=";
+  };
+
+  prePatch = ''
+    substituteInPlace src/sql_pg.c \
+       --replace-fail "#include <postgresql/libpq-fe.h>" "#include <${postgresql}/include/libpq-fe.h>"
+    substituteInPlace doc/CMakeLists.txt \
+                        --replace-fail "share/doc/gvm/html/" "\''${GMP_DIR}/doc/gvm/html/" \
```

Comment:
```suggestion
                        --replace-fail "share/doc/gvm/html/" "\''${GMP_DIR}/doc/gvm/html/" \
```
Are you indentation here is correct?

Reply:
yes it is correct,

Reply:
No, it is not:

![image](https://github.com/user-attachments/assets/a9d66059-b0d9-4de2-907e-102825017028)

Lots of extra chars. Align by 2 spaces. It should be `even`, not `odd`.

Reply:
It is still `odd`, it should be `even`:

![image](https://github.com/user-attachments/assets/f6764ecd-e8ba-4a52-a154-c60e551f939f)

This is going to error in `editorconfig-checker` and I don't want to introduce more debt.

Reply:
Please run:

> editorconfig-checker pkgs/by-name/gv/gvmd/package.nix

(And then you better understand what you are doing.)